src/api/__tests__/Feed.test.js

```diff
@@ -239,6 +239,111 @@ jest.mock('../Comments', () =>
     })),
 );
 
+jest.mock('../ThreadedComments', () =>
+    jest.fn().mockImplementation(() => ({
+        getComments: jest.fn().mockReturnValue({
+            entries: [
```

Comment:
I've left previous feedback about this topic: Can we move these long mocks into a separate file so the test suite is easier to read?